### ISSUE TITLE
Signals Connected to Method doesn't update

### ISSUE THREAD

**Issue by henriiquecampos** [godotengine/godot#33941]
**Godot version:**
3.2.beta2

**OS/device including version:**
Solus 4 Gnome

**Issue description:**

When disconnecting signals from a method using the editor interface they still show up in the **signals connected to method** interface. Even after saving the scene. They only update after reopening the scene.

![image](https://user-images.githubusercontent.com/10171059/69729607-60841900-1105-11ea-866d-8e3a14a01423.png)

![image](https://user-images.githubusercontent.com/10171059/69729679-890c1300-1105-11ea-8afa-1740b134f81d.png)

**Steps to reproduce:**

- Create a new scene
- Add some Nodes (2+)
- Attrach a script to one of them
- Connect the signal from the other Node to the Node with a script
- Check the **Connections to Method** interface
- Disconnect the signal
- Save
- Check the **Connections to Method** interface again

**Comment by Paulb23** [comment]
Possibly related #30901

**Comment by codecustard** [comment]
When you say "Check the **Connections to Method**" interface, do you mean going opening the script, then pressing on the green icon next to the signal method? When I do that, after disconnecting the signal, the green icon doesn't exist, preventing me from opening the **Connections to Method** interface.

**Comment by henriiquecampos** [comment]
> When you say "Check the **Connections to Method**" interface, do you mean going opening the script, then pressing on the green icon next to the signal method? When I do that, after disconnecting the signal, the green icon doesn't exist, preventing me from opening the **Connections to Method** interface.

Yeah I mean that exactly. Try to keep at least 1 signal connected then, like: connect 4 signals to a method, disconnect 3 then check the connections to method

**Comment by codecustard** [comment]
The issue might have something to do with your project.

I tested with the current master build as well as 3.2beta2. The signal list would update appropriately.

**Comment by henriiquecampos** [comment]
@codecustard could it be like...something to do with inherited scenes?

Like, if the scene it inherits from has signals connected to the method, and in the inherited scene I disconnect them, then the **Connections to Method** will show them as connected (because they are connected in the parent scene)?

**Comment by codecustard** [comment]
Possibly, could you try to recreate the issue in an empty project and post it here?

**Comment by KoBeWi** [comment]
Still partially valid in 3.2.3
When you disconnect signal, the green icon stays, but the connections dialog is empty. The icon disappears after you save the scene.

Same in 9e7348f
